app/i18n/en.ts

```diff
@@ -37,6 +37,7 @@ const en = {
     uniqueToPortland: "Unique to Portland",
     openInMaps: "Open in maps",
     website: "Website",
+    wantToSeeMore: "Want to see more places? Open this map!",
```

Comment:
Any way we can shorten this text? (e.g. View our destinations map)
<img width="912" alt="image" src="https://user-images.githubusercontent.com/151139/233373324-b30ece9d-5f67-4c47-9923-8193d20f95a4.png">

Reply:
after convo with gant, we're going with the `reversed` preset. screenshot updated and code updated here: https://github.com/infinitered/ChainReactApp2023/pull/150/commits/ab070d2b8f5066983b57fbaa95463131787313ef